devito/propagator.py

```diff
@@ -396,8 +402,8 @@ def generate_space_loops_blocking(self, loop_body):
             loop_limits = self._space_loop_limits[spc_var]
 
             if block_size is not None:
+                upper_limit_str = block_var + "+" + orig_var + "block"
```

Comment:
I think we should use string templates *always* because it really helps understanding what's being crafted

Like: ``"%s+%sblock"``